packages/protocol/contracts/L1/tiers/TierProviderBase.sol

```diff
@@ -57,6 +57,17 @@ abstract contract TierProviderBase is ITierProvider {
             });
         }
 
+        if (_tierId == LibTiers.TIER_ZKVM_RISC0) {
+            return ITierProvider.Tier({
+                verifierName: LibStrings.B_TIER_ZKVM_RISC0,
+                validityBond: 250 ether, // TKO
+                contestBond: 1640 ether, // =500TKO * 6.5625
```

Comment:
I have some confusion about that 500 * 6.5625 is not equal to 1640.
@dantaik @davidtaikocha

Reply:
Let me check